release-repo-scripts/bump_dependency.bash

```diff
@@ -19,10 +19,10 @@
 # Requires the 'gh' CLI to be installed.
 #
 # Usage:
-# $ ./bump_dependency.bash <collection> <library>;<library> <version>;<version> <issue_number> <prev_collection> <docs_branch>
+# $ ./bump_dependency.bash <collection> <library>;<library> <version>;<version> <issue_number> <prev_collection> [<docs_branch>]
 #
 # For example, to bump to `ign-rendering6` and all its dependencies, as well as
-# `sdf12` and its dependencies on fortress:
+# `sdf12` and its dependencies on fortress using the default `master` branch for `docs`:
```

Comment:
Does the command below actually use the default `master` branch for `docs`? The command below has `"chapulina/fortress"` as the `docs_branch` argument

Reply:
yeah you're right; I'll fix that

Reply:
https://github.com/ignition-tooling/release-tools/pull/557/commits/e6408ec28f13ccacfb178c9814728cc728bb0078